cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -926,6 +926,8 @@
       public static final String ENABLED = "security.authorization.enabled";
       /** Extension jar path */
       public static final String EXTENSION_JAR_PATH = "security.authorization.extension.jar.path";
+      /** Extra classpath for security extension **/
+      public static final String EXTENSION_EXTRA_CLASSPATH = "security.authorization.extension.extra.classpath";
```

Comment:
So now we have both classpath and jar path?? Seems super confusing. My original proposal is to keep the classpath and deprecate the jar path (meaning if both are specified, only use the classpath and ignore the jar path).